applications/StructuralMechanicsApplication/custom_processes/compute_moment_of_inertia_process.h

```diff
@@ -0,0 +1,249 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Philipp Bucher
+//                   Salman Yousaf
+//
+
+#if !defined(KRATOS_COMPUTE_MOMENT_OF_INERTIA_PROCESS)
+#define KRATOS_COMPUTE_MOMENT_OF_INERTIA_PROCESS
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "processes/process.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+/**
+ * @class ComputeMomentOfInertiaProcess
+ *
+ * @ingroup StructuralMechanicsApplication
+ *
+ * @brief This method computes the moment of inertia (rotational)
```

Comment:
I don't understand with this if you mean a rigid solid mass inertia moment or the geometrical inertia moment ?

Reply:
from the internet:
Rotational inertia is a property of any object which can be rotated. It is a scalar value which tells us how difficult it is to change the rotational velocity of the object around a given rotational axis.

Reply:
OK, so now I understand, you must specify, to differentiate between the Area Moment of Inertia or the Mass Moment of Inertia (you are computing this one)

Reply:
Ah ok now I understand your question

Yes is the mass-related one

Maybe I should change the name if it is confusing

However if you google it it only shows the mass-related stuff